src/Czechia/README.md

```diff
@@ -61,45 +98,9 @@ if (response.IsSuccess)
 - More examples are presented [here](doc/examples.md).
 - Some of the data items are explained [here](doc/data.md).
 
-# NuGet
-
-We have published the library as [Mews.Fiscalizations.Czechia](https://www.nuget.org/packages/Mews.Fiscalizations.Czechia/).
-
-Alternatively, the NuGet package can be generated by running the following command:
-
-```
-src\Mews.Fiscalizations.Czechia> nuget pack .\Mews.Fiscalizations.Czechia.csproj -IncludeReferencedProjects
-```
-
-# Tests
-Test are not currently running on Travis CI, as their network infrastructure is blocking HTTPS traffic directed into Europe, where the EET servers are located.
-
-Otherwise, a xUnit test suite is a part of the library, it contains end-to-end tests that communicates with Playground EET servers to verify the lib is really working against the current EET version.
-
-# Authors
+## 🧑 Authors
 Development: [@jirihelmich](https://github.com/jirihelmich)
 Code review: [@siroky](https://github.com/siroky), [@onashackem](https://github.com/onashackem)
 
-# Who uses the library in production?
-- [Mews](https://mews.com) - Property Management Solution for the 21st century.
-- [Hlídač EET](http://hlidaceet.cz) - Watchdog of the EET API endpoint and related services.
-- Keszh, Neszh - [Hart](http://hartphp.com.pl/) Internal ERP System
-- [Ankerdata EasiPOS](http://easipos.ankerdata.com/) - POS system
-- [sySOft](http://www.sysoft.cz/) - POS system
```

Comment:
I actually like this part 😄

Reply:
We cannot confirm or deny if that's true :D

Reply:
Well somebody did in the past (Jirka). And I believe him until proven incorrect 😂